packages/contracts-bedrock/package.json

```diff
@@ -67,7 +67,7 @@
     "@foundry-rs/easy-foundryup": "^0.1.3",
     "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@nomiclabs/hardhat-waffle": "^2.0.0",
-    "@rari-capital/solmate": "https://github.com/rari-capital/solmate.git#8f9b23f8838670afda0fd8983f2c41e8037ae6bc",
+    "@rari-capital/solmate": "https://github.com/transmissions11/solmate.git#8f9b23f8838670afda0fd8983f2c41e8037ae6bc",
```

Comment:
Here I changed rari-capital to the repo that rari-capital redirects to if you go to https://github.com/rari-capital/solmate.git#8f9b23f8838670afda0fd8983f2c41e8037ae6bc .  This was causing an issue with pnpm correctly cloning submodules within solmate